conda_devenv/devenv.py

```diff
@@ -419,7 +409,7 @@ def __call_conda_env_update(args, output_filename):
 
     old_argv = sys.argv[:]
     try:
-        sys.argv = command[1:] if env_manager == "conda" else command
+        sys.argv = command if env_manager == "mamba" else command[1:]
```

Comment:
Weird, why is this needed?

Reply:
Because ``mamba`` expects the first argument to be ``mamba``
:grimacing:

Reply:
https://github.com/TheSnakePit/mamba/blob/77c0e3cf6284aa18be0d2bfbe5ef7c2de7b642d8/mamba/mamba.py#L761

Reply:
actually, it can be anything, so I just let the full command